### PR TITLE
Fix warnings because of missing aria-label on NcButton component

### LINKED ISSUE #8247: Button lack aria labels
### Steps to reproduce

1. Open the dev console
2. Open the app

### Expected behavior

Calm logs

### Actual behavior

Bildschirmfoto vom 2023-03-21 08-35-02

### Mail app version

3

### Mailserver or service

_No response_

### Operating system

_No response_

### PHP engine version

None

### Web server

None

### Database

None

### Additional info

_No response_

### PR DESCRIPTION
**Pull request by JohannesGGE**
Fix #8247

Add aria-label attribute to avoid warning of missing.

(Hopefully any appearance of NcButton is covert)

### CODE REVIEW

File: src/components/AliasSettings.vue [hunk 1]
```diff
@@ -80,6 +86,7 @@
 			<ButtonVue v-if="showForm"
 				type="tertiary-no-background"
 				class="button-text"
+				:aria-label="t("mail", "Cancel")"
```
Comment (ChristophWurst) [inline src/components/AliasSettings.vue:89]:
```suggestion
				:aria-label="t('mail', 'Cancel')"
```

**Review by ChristophWurst — COMMENTED** [review]
Looks good otherwise!

File: src/components/AliasForm.vue [hunk 2]
```diff
@@ -58,6 +59,7 @@
 
 				<NcButton v-if="enableUpdate"
 					type="tertiary-no-background"
+					:aria-label="t('mail', 'Show update alias form')"
```
Comment (ChristophWurst) [inline src/components/AliasForm.vue:62]:
```suggestion
					:aria-label="t('mail', 'Rename alias')"
```

File: src/components/CertificateSettings.vue [hunk 3]
```diff
@@ -39,7 +39,11 @@
 			label="label"
 			track-by="id"
 			@select="selectCertificate" />
-		<Button type="primary" :disabled="certificate === null" @click="updateSmimeCertificate">
+		<Button
+			type="primary"
+			:disabled="certificate === null"
+			aria-label="Update Certificate"
+			@click="updateSmimeCertificate">
 			Update Certificate
```
Comment (ChristophWurst) [inline src/components/CertificateSettings.vue:47]: unrelated to your change but we are lacking translation here

File: src/components/CertificateSettings.vue [hunk 4]
```diff
@@ -39,7 +39,11 @@
 			label="label"
 			track-by="id"
 			@select="selectCertificate" />
-		<Button type="primary" :disabled="certificate === null" @click="updateSmimeCertificate">
+		<Button
+			type="primary"
+			:disabled="certificate === null"
+			aria-label="Update Certificate"
```
Comment (ChristophWurst) [inline src/components/CertificateSettings.vue:45]: missing translation

File: src/components/AliasForm.vue (hunk 2 above)
Comment (JohannesGGE) [reply src/components/AliasForm.vue]: :heavy_check_mark:

File: src/components/CertificateSettings.vue (hunk 3 above)
Comment (JohannesGGE) [reply src/components/CertificateSettings.vue]: :heavy_check_mark:

File: src/components/CertificateSettings.vue (hunk 4 above)
Comment (JohannesGGE) [reply src/components/CertificateSettings.vue]: :heavy_check_mark:

**Review by ChristophWurst — APPROVED** [review]

**Comment by ChristophWurst** [comment]
Please squash your commits into one, then we :shipit:

**Pull request merged**